charuco_detection:
  ros__parameters:
    camera_topic: "/camera/camera/color/image_raw"
    camera_info_topic: "/camera/camera/color/camera_info"

handeye_calibration:
  ros__parameters:
    base_frame: "base_link"
    ee_frame: "tool0"
    camera_frame: "camera_link"
